complement/pipeline.yml

```diff
@@ -62,7 +62,57 @@ steps:
           mount-buildkite-agent: false
           # Complement needs to know if it is running under CI
```

Comment:
duplicate comment... I feel like I've said this before somewhere.

Reply:
Ah, yes. I removed one instance, but there was another.